registry/txt.go

```diff
@@ -94,6 +95,7 @@ func (im *TXTRegistry) Records() ([]*endpoint.Endpoint, error) {
 			return nil, err
 		}
 		endpointDNSName := im.mapper.toEndpointName(record.DNSName)
+		log.Debug(fmt.Sprintf("Parsed endpoint dns name is %s", endpointDNSName))
```

Comment:
Please use log.Debugf so that you can omit fmt.Sprintf. Same for the lines below

Reply:
done